extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -486,6 +494,37 @@ export class AmpIframe extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Listen for Pym.js messages for 'height' and 'width'.
+   *
+   * @see http://blog.apps.npr.org/pym.js/
+   * @param {!MessageEvent} event
+   * @private
+   */
+  listenForPymMessage_(event) {
+    if (!this.iframe_ || event.source !== this.iframe_.contentWindow) {
+      return;
+    }
+    const data = getData(event);
+    if (typeof data !== 'string' || 'pym' !== data.substr(0, 3)) {
+      return;
+    }
+
+    // The format of the message takes the form of `pym${id}xPYMx${type}xPYMx${message}`.
```

Comment:
The format has a "xPYMx" between "pym` and `${id}`, too. That was confusing me since you were splicing 2 elements from the front of the array (without the "xPYMx" there, you would be splicing too much).

Reply:
Good catch. Fixed in eafb57a5891f57aa849725d0fab520f13bfc330a